static/templates/task/base.html

```diff
@@ -1,4 +1,4 @@
-<div class="_main-content" style="width: 100%">
```

Comment:
I think you just make the changes on your local branch on your machine, then `git push` and the PR should update automatically. @dmorina correct me if I'm wrong.